x-pack/plugins/security_solution/server/endpoint/endpoint_app_context_services.ts

```diff
@@ -47,41 +49,38 @@ import type {
   EndpointFleetServicesFactoryInterface,
   EndpointInternalFleetServicesInterface,
 } from './services/fleet/endpoint_fleet_services_factory';
+import { EndpointFleetServicesFactory } from './services/fleet/endpoint_fleet_services_factory';
 import { registerListsPluginEndpointExtensionPoints } from '../lists_integration';
 import type { EndpointAuthz } from '../../common/endpoint/types/authz';
 import { calculateEndpointAuthz } from '../../common/endpoint/service/authz';
 import type { FeatureUsageService } from './services/feature_usage/service';
 import type { ExperimentalFeatures } from '../../common/experimental_features';
 import type { ProductFeaturesService } from '../lib/product_features_service/product_features_service';
 import type { ResponseActionAgentType } from '../../common/endpoint/service/response_actions/constants';
+
 export interface EndpointAppContextServiceSetupContract {
   securitySolutionRequestContextFactory: IRequestContextFactory;
   cloud: CloudSetup;
   loggerFactory: LoggerFactory;
+  httpServiceSetup: HttpServiceSetup;
 }
 
 export interface EndpointAppContextServiceStartContract {
-  fleetAuthzService?: FleetStartContract['authz'];
-  createFleetFilesClient: FleetStartContract['createFilesClient'];
-  createFleetActionsClient: FleetStartContract['createFleetActionsClient'];
-  logger: Logger;
-  endpointMetadataService: EndpointMetadataService;
-  endpointFleetServicesFactory: EndpointFleetServicesFactoryInterface;
-  manifestManager?: ManifestManager;
+  fleetStartServices: FleetStartContract;
+  manifestManager: ManifestManager;
   security: SecurityServiceStart;
   alerting: AlertsPluginStartContract;
   config: ConfigType;
-  registerIngestCallback?: FleetStartContract['registerExternalCallback'];
   registerListsServerExtension?: ListsServerExtensionRegistrar;
   licenseService: LicenseService;
   exceptionListsClient: ExceptionListClient | undefined;
   cases: CasesServerStart | undefined;
   featureUsageService: FeatureUsageService;
   experimentalFeatures: ExperimentalFeatures;
-  messageSigningService: MessageSigningServiceInterface | undefined;
+  /** An internal ES client */
   esClient: ElasticsearchClient;
```

Comment:
Can the name specify that it's an internal ES client?

Reply:
I rather keep it as is, since its consistent with how we normally name the ES client variables through out our code base. Is that cool with you?

Reply:
Sure 👍  thanks!